app/src/sharedTest/java/org/oppia/app/help/HelpFragmentTest.kt

```diff
@@ -55,6 +57,29 @@ class HelpFragmentTest {
     }
   }
 
+  @Test
+  fun openHelpActivity_configurationChanged_scrollRecyclerViewToZeroPosition_showsFAQSuccessfully() {
+    launch(HelpActivity::class.java).use {
+      onView(isRoot()).perform(orientationLandscape())
+      onView(withId(R.id.help_fragment_recycler_view)).perform(
+        scrollToPosition<RecyclerView.ViewHolder>(
+          0
+        )
+      )
+      onView(
+        atPositionOnView(
+          R.id.help_fragment_recycler_view,
+          0,
+          R.id.recycler_item_text_view
+        )
+      ).check(
+        matches(
+          withText(R.string.frequently_asked_questions_FAQ)
```

Comment:
Let's fix this also. Also, rename `frequently_asked_questions_FAQ ` to `frequently_asked_questions` in `strings.xml` file and wherever else it is getting used.

Reply:
There already exists a  string with frequently_asked_questions maybe that's why someone used _FAQ to distinguish  between the two